tests/subtests/test_subtest.py

```diff
@@ -4,13 +4,15 @@
 import subprocess
 from pathlib import Path
 
-import matplotlib
 import matplotlib.ft2font
 import pytest
 from packaging.version import Version
 
 from .helpers import assert_existence, diff_summary, patch_summary
 
+# TODO: Remove this skip, see issue https://github.com/matplotlib/pytest-mpl/issues/159
+pytest.skip(reason="temporarily disabled sub-tests", allow_module_level=True)
+
```

Comment:
@ConorMacBride I'll remove this and rebase when #163 is merged 👍

Reply:
Thanks! Out of curiosity, I tried that and it generates the baseline data correctly. The only modification that's needed to the baseline generation functions is https://github.com/matplotlib/pytest-mpl/pull/163#discussion_r884159274